packages/controllers/src/snaps/SnapController.test.ts

```diff
@@ -254,6 +255,35 @@ const getTruncatedSnap = ({
   } as const;
 };
 
+jest.mock('./utils', () => ({
+  ...(jest.requireActual('./utils') as any),
```

Comment:
Doesn't this return `any` by default? Even better would be to specify the type, e.g. `jest.requireActual<typeof utils>('./utils')`.